modules/primo/src/Primo/BriefSearch/Client.php

```diff
@@ -109,6 +119,12 @@ public function search($query, $offset, $numResults, $parameters = []) {
       'indx' => $offset,
       'bulkSize' => $numResults,
     ] + $this->defaultParameters;
+
+    // If we're configured to search in a scope, add it.
+    if (!empty($this->locationScopes)) {
+      $scopes_string = implode(',', $this->locationScopes);
+      $parameters['loc'] = 'local,scope:(' . $scopes_string . ')';
+    }
```

Comment:
In effect this will prevent callers from setting `loc` as a part of `$parameters` here as it will be overridden subsequently.

Is there a reason why the conversion from `$locationScopes` to the `loc` (default) parameter is not done by the constructor?

Reply:
Good idéa to do it in the constructor.

Yes, it will prevent others from setting loc. I suggest we delay fixing this a bit as I shortly will run in to this issue from another angle when we have to support adding multiple query parameters to the URL which Guzzle (as far as I can tell) don't support doing using the array-approach for specifying parameters.

To get multiple queries up and running We'll probably have to go for a nested 
[ [ 'key' => 'value'], ['key' => 'value']] approach for the parameters at which point I'll make sure to support multiple loc-parameters.

Makes sense?

Reply:
I am not 100% sure I understand.

I think we should do our best to remove any weirdness caused by the inner workings of the PrimoClient from the external interface.

In other words: If Guzzle does not support something then we should try to convert from sane input to the Guzzle way within the client itself.